graphql/config_resolver.go

```diff
@@ -2,11 +2,38 @@ package graphql
 
 import (
 	"context"
+	"fmt"
 	"sort"
 
+	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/utility"
 )
 
+// BannerTheme is the resolver for the bannerTheme field.
+func (r *adminSettingsResolver) BannerTheme(ctx context.Context, obj *model.APIAdminSettings) (*evergreen.BannerTheme, error) {
+	if obj == nil {
+		return nil, InternalServerError.Send(ctx, "Banner theme undefined when attempting to resolve communication method")
+	}
+
+	bannerTheme := evergreen.BannerTheme(utility.FromStringPtr(obj.BannerTheme))
+
+	switch bannerTheme {
+	case evergreen.Announcement:
+		return &bannerTheme, nil
+	case evergreen.Information:
+		return &bannerTheme, nil
+	case evergreen.Warning:
+		return &bannerTheme, nil
+	case evergreen.Important:
+		return &bannerTheme, nil
+	case evergreen.Empty:
+		return &bannerTheme, nil
+	default:
+		return nil, InputValidationError.Send(ctx, fmt.Sprintf("Banner theme '%s' is invalid", utility.FromStringPtr(obj.BannerTheme)))
+	}
```

Comment:
I'd probably recommend something like this:
```suggestion
	if obj == nil {
		return nil, InternalServerError.Send(ctx, "admin settings object undefined when attempting to resolve banner theme")
	}
	themeString := strings.ToUpper(utility.FromStringPtr(obj.BannerTheme))
	valid, theme := evergreen.IsValidBannerTheme(themeString)
	if !valid {
		return nil, InputValidationError.Send(ctx, fmt.Sprintf("invalid banner theme '%s'", themeString))
	}
	return &theme, nil
```